### ISSUE TITLE
Organization repos

### ISSUE THREAD

**Issue by jtrumbull** [mirumee/overv.io-beta#521]
**Firstly**: Awesome! Was waiting to find a BaseCamp like app with GitHub integration.

My GitHub account owns a couple organizations, which I primarily work out of. I went to _Create a new Project_, but did not see my organization repos listed under the _Select your repositories_ dropdown. Are organizations supported in **_overv.io**_?

**Comment by patrys** [comment]
Yes, we do support organizations! Not supporting them would be a bit on the silly side. A little while ago GitHub added support for [organizations manually approving application access](https://github.com/blog/1941-organization-approved-applications). Could it be that your organization has this feature (rightfully!) enabled? If so, please have an organization admin approve overv.io and you should be able to access your repos just fine.

**Comment by mirekm** [comment]
Apart from that we've also found a small problem there affecting some of the users. The appropriate hotfix is now online so please check again and let us now if there's any change.
Thanks for the feedback!
